sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/SpringSdkMessageCodec.scala

```diff
@@ -31,10 +32,14 @@ class SpringSdkMessageCodec extends MessageCodec {
    * In the Spring SDK, output data are encoded to Json.
    */
   override def encodeScala(value: Any): any.Any =
-    any.Any.fromJavaProto(JsonSupport.encodeJson(value))
+    any.Any.fromJavaProto(encodeJava(value))
+
+  override def encodeJava(value: Any): protobuf.Any =
+    JsonSupport.encodeJson(value)
```

Comment:
Boyscouting: Perhaps good to go with the pattern we've used in the proxy where we import scala any with an alias `ScalaPbAny` and the Java any `JavaPbAny` for extra clarity?